app/AndroidManifest.xml

```diff
@@ -494,9 +494,6 @@
 
         <activity android:name="org.commcare.activities.CommcareListActivity"/>
 
-        <meta-data android:name="android.content.APP_RESTRICTIONS"
```

Comment:
is this no longer needed ?

Reply:
This is a duplicate entry. https://github.com/dimagi/commcare-android/blob/master/app/AndroidManifest.xml#L490 and https://github.com/dimagi/commcare-android/blob/master/app/AndroidManifest.xml#L497. 
So I removed one of them after noticing the warning about the duplicate entries.